pkg/status/combinedstatus.go

```diff
@@ -83,6 +87,13 @@ func (c *Controller) updateOrCreateCombinedStatus(ctx context.Context, combinedS
 	// set results
 	combinedStatus.Results = generatedCombinedStatus.Results
```

Comment:
`combinedStatus` points to storage fetched from the informer's local cache, which is supposed to be immutable, right?